src/cmd/deploy.ts

```diff
@@ -118,6 +124,17 @@ export default class DefenderDeploy {
       (a: DefenderContract, b: [string, Contract]) => `${a.network}-${a.address}` === `${b[1].network}-${b[1].address}`,
     );
 
+    // Forked Networks
+    const forkedNetworks: ForkedNetworks = this.resources?.['forked-networks'] ?? {};
+    const networkClient = getNetworkClient(this.teamKey!);
+    const forkedNetworkItems = await networkClient.listForkedNetworks();
+    const forkedNetworkDifference = _.differenceWith(
+      forkedNetworkItems,
+      Object.entries(forkedNetworks),
+      (a: DefenderForkedNetwork, b: [string, ForkedNetworkRequest]) =>
+        a.stackResourceId === getResourceID(getStackName(this.serverless), b[0]),
+    );
```

Comment:
If the user just wants to change the api key or explorer url, must also change the id in order to impact the app, right? I think we could clarify that in docs